.gitignore

```diff
@@ -8,3 +8,4 @@ MANIFEST
 /.tox/
 /build/
 /dist/
+venv
```

Comment:
I don't believe this is part of any repo-specific tooling, so if this is a workflow you use across repos (e.g. how you name all virtualenvs), I would also just recommend that you create a global `.gitignore` file using e.g. something like this in your `~/.gitconfig`:

```
[core]
	excludesfile = ~/.gitignore.global
```

Reply:
Ah, didn't realize this was possible. Done!